src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -771,22 +772,40 @@ public DatasetSummaryModel updateTags(UUID id, TagUpdateRequestModel tagUpdateRe
     return datasetDao.retrieveSummaryById(id).toModel();
   }
 
+  public static boolean isInheritedRole(IamRole role) {
+    return isInheritedRole(role.toString());
+  }
+
+  public static boolean isInheritedRole(String roleName) {
```

Comment:
Too bad we need the `String` version here, I see that's because it's used for filtering `SamPolicyModel`.